.github/workflows/ci.yaml

```
@@ -0,0 +1,102 @@
+name: deploy-probot-terraform
+
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches:
+      - "pull-request/[0-9]+"
+      - "main"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

```suggestion
  cancel-in-progress: false
```

Canceling in-progress `terraform apply` commands will cause issues with state locking. Therefore this should be set to `false` to avoid those types of issues.